test/functional/tools/output_format_collection.xml

```diff
@@ -0,0 +1,35 @@
+<tool id="output_format_collection" name="output_format_collection" version="1.0.0">
+  <command>
+    echo "test" > 1;
+    echo "test" > 2;
+  </command>
+  <inputs>
+    <param name="input_collection" type="data_collection" format="data" />
+  </inputs>
+  <outputs>
+    <!-- Access by element name (for paired data) -->
+    <data format="txt" from_work_dir="1" name="format_source_1_output" format_source="input_collection['forward']" />
```

Comment:
I think `format="txt"` here and below is hiding the fact the new code doesn't work :cry:.
If I removed them, I got "Dataset metadata verification for [file_ext] failed, expected [txt] but found [data]."
Similarly with `fastqsanger` in sickle.